bundles/AdminBundle/Resources/public/js/pimcore/object/tags/manyToManyObjectRelation.js

```diff
@@ -342,9 +342,11 @@ pimcore.object.tags.manyToManyObjectRelation = Class.create(pimcore.object.tags.
                 fc.editor = null;
                 fc.sortable = false;
 
-                if(fc.layout.key === "fullpath") {
+                if (fc.layout.key === "fullpath") {
                     fc.renderer = this.fullPathRenderCheck.bind(this);
-                } else if(fc.layout.layout.fieldtype == "select" || fc.layout.layout.fieldtype == "multiselect") {
+                } else if (fc.layout.layout.fieldtype == 'select'
+                    || fc.layout.layout.fieldtype == 'multiselect'
+                    ||fc.layout.layout.fieldtype == 'booleanSelect') {
```

Comment:
There should be a space between `||` and fc

Reply:
true i must have missed this one - thanks for commenting